packages/gatsby-plugin-bumbag/gatsby-ssr.js

```diff
@@ -6,10 +6,10 @@
 
 // You can delete this file if you're not using it
 
-import React from 'react';
-import { renderToString } from 'react-dom/server';
-import { renderStylesToString } from 'emotion-server';
-import { InitializeColorMode } from 'bumbag';
+import React from "react";
```

Comment:
Can we format this file?

Reply:
I think this is left out from the `yarn lint:fix` script, but fixed manually